apps/frontend/src/libs/components/balance-wheel-chart/libs/helpers/draw-extra-point-graphics/draw-extra-point-graphics.helper.ts

```diff
@@ -0,0 +1,58 @@
+import { TAU, ZERO_INDEX } from "~/libs/constants/constants.js";
+import {
+	type Chart,
+	type ChartDataset,
+	type RadialLinearScale,
+} from "~/libs/types/types.js";
+
+import { RADIAL_AXIS_SCALE } from "../../constants/constants.js";
+import { type PolarAreaType } from "../../types/types.js";
+import { drawDots } from "../draw-dots/draw-dots.helper.js";
+import { drawSublabels } from "../draw-sublabels/draw-sublabels.helper.js";
+import { drawValueLabels } from "../draw-value-labels/draw-value-labels.helper.js";
+import { getMiddleAngle } from "../get-middle-angle/get-middle-angle.helper.js";
+
+const drawExtraPointGraphics = (chart: Chart<PolarAreaType>): void => {
+	const { ctx: context } = chart;
+	const scale = chart.scales[RADIAL_AXIS_SCALE] as RadialLinearScale;
+	const meta = chart.getDatasetMeta(ZERO_INDEX);
+	const { chartArea } = chart;
+
+	const sublabels = chart.data.labels as string[];
+	const dataset = chart.data.datasets[ZERO_INDEX] as ChartDataset<
+		PolarAreaType,
+		number[]
+	>;
+	const values = dataset.data;
+
+	const angleStep = TAU / sublabels.length;
+
+	for (const [index, value] of values.entries()) {
+		const middleAngle = getMiddleAngle(index, angleStep);
+		const label = String(value);
+
+		drawValueLabels({
+			chartArea,
+			context,
+			label,
+			middleAngle,
+			scale,
+		});
+	}
+
+	for (const [index, label] of sublabels.entries()) {
+		const middleAngle = getMiddleAngle(index, angleStep);
+
+		drawSublabels({
+			chartArea,
+			context,
+			label,
+			middleAngle,
+			scale,
+		});
+
+		drawDots({ chartArea, context, index, meta, middleAngle, scale });
+	}
```

Comment:
Let's use forEach() cycle here. Your solution is good but forEach will make it more "JS like" solution

Reply:
```values.entries().forEach(([index, label]) => {})```
and
```sublabels.entries().forEach(([index, label]) => {})```

Reply:
Eslint shows me an error when I am trying to use forEach here
```
Use `for…of` instead of `.forEach(…)`.eslint[unicorn/no-array-for-each](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v55.0.0/docs/rules/no-array-for-each.md)
```

Reply:
Okey, then it is fine to leave for of cycle